app/javascript/sdk/IFrameHelper.js

```diff
@@ -140,6 +146,9 @@ export const IFrameHelper = {
         IFrameHelper.pushEvent('webwidget.triggered');
       }
     },
+    onLocationChange: url => {
+      IFrameHelper.sendMessage('change-url', { url });
```

Comment:
There is an event `set-current-url`. Could you please check if we can reuse that?

Reply:
The event `set-current-url` fires only when url hash changes. It doesn't work on non-hash URL changes.